### PR TITLE
Rust isn't functional, but it does have functional elements

### PR DESCRIPTION
**Pull request by domenukk**
Add information about this

### CODE REVIEW

File: src/hello-world.md [hunk 1]
```diff
@@ -37,4 +37,7 @@ Key points:
   the scope they are used in. Rust macros are actually only
   [partially hygienic](https://veykril.github.io/tlborm/decl-macros/minutiae/hygiene.html).
 
+* While it is not a functional language, it includes a range of
+  [functional concepts](https://doc.rust-lang.org/book/ch13-00-functional-features.html)
```
Comment (mgeisler) [inline src/hello-world.md:41]:
```suggestion
  [functional concepts](https://doc.rust-lang.org/book/ch13-00-functional-features.html).
```

File: src/hello-world.md [hunk 2]
```diff
@@ -37,4 +37,7 @@ Key points:
   the scope they are used in. Rust macros are actually only
   [partially hygienic](https://veykril.github.io/tlborm/decl-macros/minutiae/hygiene.html).
 
+* While it is not a functional language, it includes a range of
```
Comment (sakex) [inline src/hello-world.md:40]: I don't think we want to say that, we could make the point about any paradigm. Ie it's not OO but has aspects of object orientation

**Review by mgeisler — APPROVED** [review]

File: src/hello-world.md (hunk 2 above)
Comment (mgeisler) [reply src/hello-world.md:40]: You're right, but I've seen lots of people call Rust somewhat functional.

Could we include what you just wrote as well? 🙂 It's a common question in class! People like to try and classify Rust and it's neither a strict functional language nor is it a strict OO language.

Comment (domenukk) [reply src/hello-world.md:40]: Yes maybe add OO as well? I'm fine with dropping functional again here as well, or just saying "Mutli-Paradigm"(as wikipedia does)

Comment (domenukk) [reply src/hello-world.md]: I have added that, please take another look

**Review by sakex — APPROVED** [review]

**Pull request merged**